### PR TITLE
[RAINCATCH-1255] Add instructions for running from the separate repositories

### PR DESCRIPTION
**Pull request by paolobueno**
# Motivation
Include instructions from running the solution from the separate release repositories + public npm packages.

# Description
- [X] Add extra section on cloning the separate repos and running the demo apps from them
- [ ] Include new section in the main document

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+----
+cd raincatcher-server/
+npm run start
+----
+
+[id='pro-running-the-mobile-and-portal-demo-applications-{chapter}']
+[discrete]
+== Running the Mobile and Portal Demo Applications
+
+Open a new terminal and navigate to the folder containing _raincatcher-angularjs_, and execute:
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: Will _raincatcher-angularjs_ exist when installing individual packages?

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+This procedure will install the latest published version of {WFM-RC-NameShort} from the separate, per-demo-app GitHub repositories.
+
+[id='pro-cloning-separate-repositories-{chapter}']
+[discrete]
+== Cloning the Git Repositories
+
+[source,bash]
+----
+git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: We need to call here explicity what branch we want to use - master for recent changes. 
We may have releases that are RC1 so people should be never pointed to master.
Just current released version that will be a tag.

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+This section includes:
+
+. xref:pro-cloning-separate-repositories-{chapter}[Cloning the Git Repository]
+. xref:pro-installing-repository-npm-dependencies-{chapter}[Installing Core Dependencies]
+. xref:pro-starting-mongodb-and-redis-{chapter}[Starting MongoDB and Redis]
+. xref:pro-starting-the-core-server-{chapter}[Starting the Core Server]
+. xref:pro-running-the-mobile-and-portal-demo-applications-{chapter}[Running the Mobile and Portal Demo Applications]
+
+= Prerequisites
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: Follow aproach from https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/63 
Let's not create sections in the toc for this but just standard headings.

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc (hunk 1 above)
Comment (paolobueno) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: Thanks for catching this, forgot to remove this paragraph in favor of the next one :joy:

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc (hunk 2 above)
Comment (paolobueno) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: Done!

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc (hunk 3 above)
Comment (paolobueno) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: Done!

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc (hunk 2 above)
Comment (paolobueno) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: via 43410db3d611c8d43ff6160a825f79429e2c7fc0

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc (hunk 3 above)
Comment (paolobueno) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: via 04b294b

**Comment by paolobueno** [comment]
@austincunningham By the way I haven't added the new content in this PR to the docs/ page itself, because I wasn't sure where to put it. Should it go after `3.1. Running the Demo App` ?

**Review by austincunningham — COMMENTED** [review]
verified

**Review by austincunningham — COMMENTED** [review]
@paolobueno  Will leave the doc team decide where to put it, happy for this to go to tech approved.

**Comment by wtrocki** [comment]
As decided with @austincunningham we no longer needs that tags as we going to filter them out on the top level.

**Comment by paolobueno** [comment]
closing in favor of #69

**Comment by wtrocki** [comment]
@paolobueno cannot find this change. Looks like it's lost.

**Comment by paolobueno** [comment]
Rebased on top of current master, trying to recover these changes

ping @wtrocki @jstaffor

**Comment by wtrocki** [comment]
I will wait for @jstaffor to merge. We can generate website from this bramch

**Comment by jstaffor** [comment]
@wtrocki - you do realize that if we merge this - our agreement that no existing content is to change, is no longer applicable and downstream and upstream will diverge? If you still want me to merge this, let me know.

**Comment by wtrocki** [comment]
@jstaffor I asked Paolo to clean that up so it should be ok now.

**Comment by wtrocki** [comment]
@jstaffor Moving to separate branch. That's not merged to master.

**Pull request merged**